code/controllers/configuration/entries/game_options.dm

```diff
@@ -112,27 +112,7 @@ CONFIG_DEF(flag/show_game_type_odds)	//if set this allows players to see the odd
 
 CONFIG_DEF(flag/join_with_mutant_race)	//players can choose their mutant race before joining the game
 
-CONFIG_DEF(keyed_flag_list/roundstart_races)	//races you can play as from the get go. If left undefined the game's roundstart var for species is used
-	var/first_edit = TRUE
-
-/datum/config_entry/keyed_flag_list/roundstart_races/New()
-	for(var/I in subtypesof(/datum/species))
-		var/datum/species/S = I
-		if(initial(S.roundstart))
-			value[initial(S.id)] = TRUE
-	..()
-
-/datum/config_entry/keyed_flag_list/roundstart_races/ValidateAndSet(str_val)
-	var/list/old_val
-	if(first_edit)
-		old_val = value
-		old_val = old_val.Copy()
-	. = ..()
-	if(first_edit)
-		if(!.)
-			value = old_val
-		else
-			first_edit = FALSE
+CONFIG_DEF(keyed_flag_list/roundstart_races)	//races you can play as from the get go.

```

Comment:
you might want to protect this config, although might be way too much

Reply:
You can already do that in the .txt if need be. Save it for the sql and comms stuff

Reply:
oh yea, i forgot about it